xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-default/src/main/java/org/xwiki/mentions/internal/DefaultMentionXDOMService.java

```diff
@@ -85,12 +86,13 @@ private static boolean matchMentionMacro(Block block)
     }
 
     @Override
-    public Map<DocumentReference, List<String>> countByIdentifier(List<MacroBlock> mentions)
+    public Map<DocumentReference, List<String>> groupAnchorsByUserReference(List<MacroBlock> mentions,
+        WikiReference wikiReference)
     {
         Map<DocumentReference, List<String>> ret = new HashMap<>();
         for (MacroBlock block : mentions) {
             String macroReference = block.getParameter(REFERENCE_PARAM_NAME);
```

Comment:
This variable is very badly name if it's supposed to be the serializedUserReference.